### PR TITLE
Demonstrate put_in to manipulate nested struct values

### PR DESCRIPTION
**Pull request by srufle**
#161

### CODE REVIEW

**Comment by felipesere** [comment]
Are we covering the most common cases of put_in and friends? I haven't used them before. 
Also, add yourself to the CONTRIBUTORS.md 🙂

File: lib/koans/11_structs.ex [hunk 1]
```diff
@@ -59,4 +59,14 @@ defmodule Structs do
     airline = %Airline{plane: %Plane{maker: :boeing, passengers: 200}}
     assert update_in(airline.plane.passengers, &(&1 + 2)) == ___
   end
+
+  koan "Use the put_in macro with atoms to replace a nested value in a non-struct" do
+    airline = %{
```
Comment (iamvery) [inline lib/koans/11_structs.ex]: Maybe one-line this map like the structs in the above tests?

Comment (srufle) [inline lib/koans/11_structs.ex]: K, I will put it all on one line. I make things multiline so I can see teh brace matches easier.

Comment (iamvery) [inline lib/koans/11_structs.ex]: That makes sense to me! I was just thinking it should be consistent across tests (either one line or multi line)

**Comment by felipesere** [comment]
Sorry I forgot to merge this. Thank you so much! ❤️

**Pull request merged**